FIAT/quadrature_schemes.py

```diff
@@ -91,76 +103,313 @@ def _fiat_scheme(ref_el, degree):
     return make_quadrature(ref_el, num_points_per_axis)
 
 
+def _kmv_lump_scheme(ref_el, degree):
+    """Specialized quadrature schemes for P < 6 for KMV simplical elements."""
+
+    sd = ref_el.get_spatial_dimension()
+    # set the unit element
+    if sd == 2:
+        T = UFCTriangle()
+    elif sd == 3:
+        T = UFCTetrahedron()
+
+    if degree == 1:
+        x = ref_el.vertices
+        w = arange(sd + 1, dtype=float64)
+        if sd == 2:
+            w[:] = 1.0 / 6.0
+        elif sd == 3:
+            w[:] = 1.0 / 24.0
+    elif degree == 2:
+        if sd == 2:
+            x = list(ref_el.vertices)
+            for e in range(3):
+                x.extend(ref_el.make_points(1, e, 2))  # edge midpoints
+            x.extend(ref_el.make_points(2, 0, 3))  # barycenter
+            w = arange(7, dtype=float64)
+            w[0:3] = 1.0 / 40.0
+            w[3:6] = 1.0 / 15.0
+            w[6] = 9.0 / 40.0
+        elif sd == 3:
+            x = list(ref_el.vertices)
+            x.extend(
+                [
+                    (0.0, 0.50, 0.50),
+                    (0.50, 0.0, 0.50),
+                    (0.50, 0.50, 0.0),
+                    (0.0, 0.0, 0.5000),
```

Comment:
Why more `0`s after the `5` on this line?

Reply:
Typo. Fixed.